libs/wire-api/src/Wire/API/Connection.hs

```diff
@@ -280,61 +241,37 @@ data ConnectionRequest = ConnectionRequest
   { -- | Connection recipient
     crUser :: UserId,
     -- | Name of the conversation to be created
-    crName :: Text,
-    -- | Initial message
-    crMessage :: Message
+    -- FUTUREWORK investigate: shouldn't this name be optional? Do we use this name actually anywhere?
```

Comment:
> I'm just guessing now, but does a connection name ever make it to a one-to-one conversation title?

I'm not sure, I didn't take the time to look more deeply into how this is used and what clients send and receive and what they do with it. That's why there is a FUTUREWORK here. A potential refactor that removes this can be done in a subsequent PR.